editor/find_in_files.cpp

```diff
@@ -221,8 +221,8 @@ float FindInFiles::get_progress() const {
 
 void FindInFiles::_scan_dir(String path, PoolStringArray &out_folders) {
 
-	DirAccess *dir = DirAccess::open(path);
-	if (dir == NULL) {
+	DirAccessRef dir = DirAccess::open(path);
+	if (dir) {
```

Comment:
That's wrong, `if (dir)` is like `if (dir != NULL)`, which is the opposite of the previous code. That will break the editor.

Reply:
Literally the only change needed here was to replace `DirAccess *` by `DirAccessRef`, which is equivalent but freed when out of scope, or add a `memdelete(dir)`  before any return.